workflows/raredisease.nf

```diff
@@ -26,33 +28,28 @@ def checkPathParamList = [
     params.intervals_y,
     params.known_dbsnp,
     params.known_dbsnp_tbi,
-    params.known_indels,
-    params.known_mills,
     params.ml_model,
-    params.mt_backchain_shift,
-    params.mt_bwa_index_shift,
-    params.mt_bwamem2_index_shift,
-    params.mt_fasta_shift,
-    params.mt_fai_shift,
-    params.mt_intervals,
-    params.mt_intervals_shift,
-    params.mt_sequence_dictionary_shift,
+    params.mt_fasta,
     params.multiqc_config,
     params.reduced_penetrance,
     params.score_config_snv,
     params.score_config_sv,
     params.sequence_dictionary,
-    params.target_bed,
     params.svdb_query_dbs,
+    params.target_bed,
     params.variant_catalog,
-    params.vep_filters,
     params.vcfanno_lua,
     params.vcfanno_resources,
     params.vcfanno_toml,
-    params.vep_cache
+    params.vep_cache,
+    params.vep_filters
 ]
 
 for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
```

Comment:
You can get rid of this provided you have add `"exists": true,` in the `nextflow_schema.json` for all this file parameters
```suggestion
```

Reply:
Good point @JoseEspinosa!